frontend/model/contracts/group.js

```diff
@@ -1154,6 +1160,8 @@ sbp('chelonia/defineContract', {
                     }
                   }
                 }).catch((e) => {
+                  // If already joined, ignore this error
+                  if (e?.name === 'GIErrorUIRuntimeError' && e.cause?.name === 'GIGroupAlreadyJoinedError') return
```

Comment:
Where in the code do we throw a `GIErrorUIRuntimeError` with a `cause` of `GIGroupAlreadyJoinedError`? 🤔

Reply:
This error class (`GIGroupAlreadyJoinedError `) is new, along with a few others, and is thrown in the group contract. I added this after your comment of not relying on the error message.

Reply:
The `cause` itself is set in `frontend/controller/actions/utils.js:244` (`throw new GIErrorUIRuntimeError(userFacingErrStr, { cause: e })`), where `e` may have been thrown by the contract.